core/src/main/web/app/helpers/core.js

```diff
@@ -732,31 +751,41 @@
         if (!msgHeader) {
           msgHeader = "Question...";
         }
-        var close = function(result) {
-          if (result === "Yes") {
-            yesCB ? yesCB() : null;
-          } else if (result === "No") {
-            noCB ? noCB() : null;
-          } else { // cancel
-            cancelCB ? cancelCB() : null;
+        var callback = function(result) {
+          if (((result === "Yes") || (result == 0)) && yesCB) {
```

Comment:
How do you feel about this code?

I think it can be cleaned up to be much more readable and maintainable